gatsby-node.js

```diff
@@ -1,7 +1,105 @@
-/**
- * Implement Gatsby's Node APIs in this file.
- *
- * See: https://www.gatsbyjs.com/docs/node-apis/
- */
+const path = require('path');
+const { createFilePath } = require('gatsby-source-filesystem');
+const { constructPathPrefix } = require('./src/utils/language');
 
-// You can delete this file if you're not using it
+const {
+  defaultLanguageCode,
+  languages,
+} = require('./gatsby-config').siteMetadata;
```

Comment:
@victor77dev 

Yes!! because it's run by node.js during the webpack stage.

I knew we can use `import` as well but i just skipped the configuration work orz

Reply:
So let's just continue with `require`.